src/Turn.js

```diff
@@ -0,0 +1,35 @@
+class Turn {
+    constructor(userGuess, card) {
+     this.userGuess = userGuess
+     this.card = card
+   }
+
+returnGuess() {
+    return this.userGuess
+}
+
+returnCard() {
+    console.log(this.card)
+    return this.card
+
+}
+
+evaluateGuess() {
+    if (this.userGuess === this.card.correctAnswer) {
+        return true
+    } else {
+        return false
+    }
+}
+
+giveFeedback() {
```

Comment:
This seems a little repetitive since we are checking the same thing but it returns something different, maybe I can change this method?